### PR TITLE
LPS-136360 Add new log type to workflow

### PR DESCRIPTION
**Pull request by MariMalta**
Story: [LPS-136360](https://github.com/liferay-workflow/liferay-portal/pull/479)

### CODE REVIEW

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:relevant

File: modules/apps/headless/headless-admin-workflow/headless-admin-workflow-impl/src/main/java/com/liferay/headless/admin/workflow/internal/resource/v1_0/WorkflowLogResourceImpl.java [hunk 1]
```diff
@@ -105,6 +105,9 @@ private String _toLogTypeName(WorkflowLog.Type type) {
 		else if (type == WorkflowLog.Type.TASK_COMPLETION) {
 			return LogType.TASK_COMPLETION.name();
 		}
+		else if (type == WorkflowLog.Type.NODE_ENTRY) {
```
Comment (MarinhoFeliphe) [inline modules/apps/headless/headless-admin-workflow/headless-admin-workflow-impl/src/main/java/com/liferay/headless/admin/workflow/internal/resource/v1_0/WorkflowLogResourceImpl.java:108]: @MariMalta, could you put this conditional above the `WorkflowLog.Type.TASK_ASSIGN` conditional to keep the alphabetical order?, thank you!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by MarinhoFeliphe** [comment]
Hi, @MariMalta, this [commit](https://github.com/liferay-workflow/liferay-portal/pull/479/commits/ae50c28b56b790872e10e8f0cfedc7c411ae637e) there isn't any changes, could you verify please?

**Comment by MariMalta** [comment]
> Hi, @MariMalta, this [commit](https://github.com/liferay-workflow/liferay-portal/pull/479/commits/ae50c28b56b790872e10e8f0cfedc7c411ae637e) there isn't any changes, could you verify please?

I'm going to check what happend and submit the change

**Comment by MariMalta** [comment]
ci:test:sf

**Review by rafaprax — APPROVED** [review]

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by MariMalta** [comment]
ci:forward

**Pull request closed without merging**